### DISCUSSION TITLE
Progetto di esame Danilo Palladino

### DISCUSSION THREAD

**Discussion by daniloPalladino04** [deneb-alpha/Lab_open_source-2023_2024#20] [Projects]
Essenziale:

- Progetto Localizzazione LibreOffice Help (Inglese/italiano) via Weblate
- Cosa è una community open source e cosa significa farne parte.

Non essenziale:

- Background sulla sicurezza di LibreOffice

**Comment by deneb-alpha** [comment]
Ok @daniloPalladino04,
all'esame cerca anche di descrivere non solo gli aspetti tecnici di traduzione utilizzando Weblate, ma descrivi anche come hai interagito con la community e, in particolar modo, esponi anche cosa potrebbe essere migliorato per permettere a nuovi traduttori di contribuire.

Se hai modo, anche per coprire gli argomenti più tecnici su git che abbiamo visto a lezione, potresti anche crearti un tuo repository prersonale dove caricare le slide dell'esame (se hai intenzione di farne).

**Reply by daniloPalladino04** [reply]
La ringrazio per il suggerimento prof e mi scusi se rispondo solo ora, ne prendo immediatamente atto.
Buona giornata
